shared/src/popup.css

```diff
@@ -18,6 +18,7 @@ body {
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  width: 483px;
```

Comment:
I found that this was the best width to keep things consistent between Firefox and Chromium-based browsers in the popup.